pgcli/pgexecute.py

```diff
@@ -491,6 +500,13 @@ def full_databases(self):
             headers = [x[0] for x in cur.description]
             return cur.fetchall(), headers, cur.statusmessage
 
+    def socket_directory(self):
```

Comment:
I would suggest renaming the method to `get_socket_directory`, and adding a new property `socket_directory` to pgexecute class, that would be populated if host is empty.